index.html

```diff
@@ -493,7 +493,7 @@ <h4>Workspace</h4>
                 <span class="log-open-legend-dialog glyphicon glyphicon-cog" data-toggle="tooltip" title="Show the legend"></span>
             </div>
             <div class="log-graph-config">
-                <h2>Legend
+                <h2 id="legend_title">Legend
```

Comment:
`legend_title` isn't used anywhere - should this be part of #401?

Reply:
yes, I'm sorry but I don't know how to upload two files together for one pull request :-O

Reply:
Merged this for you - you should really start checking out the repository locally when working on changes - the online editors have some nasty bugs.